test/__snapshots__/Changelog.spec.js.snap

```diff
@@ -0,0 +1,67 @@
+exports[`createMarkdown get markdown grouped by tags (with commit number link) 1`] = `
+"
+## v0.2.0 (2017-01-01)
+
+#### :rocket: New Feature
+* \`pkg-1\`, \`pkg-2\`
+  * [#1](https://github.com/lerna/lerna-changelog/pull/1) This is the commit title for the issue (#1). ([@test-user](https://github.com/test-user))
+  * [#1](https://github.com/lerna/lerna-changelog/pull/1) This is the commit title for the issue (#1). ([@test-user](https://github.com/test-user))
```

Comment:
Yeah, actually it's correct to have 2 commit entries here. The problem is that the mock returns the same value, thus you see the same message. But I want to refactor a bit more the way we're mocking stuff, making it easier to setup the tests.
I'll push some changes soon.